clusterloader2/pkg/measurement/common/network-policy/manifests/policy-egress-allow-target-pods.yaml

```diff
@@ -1,3 +1,61 @@
+{{ if eq .NetworkPolicy_Type "cnp" }}
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: {{.Name}}
+  namespace: {{.ClientNamespace}} # since this policy is egress, the namespace is the client namespace
+  labels:
+    type: {{.TypeLabelValue}}
+spec:
+  endpointSelector:
+    matchLabels:
+      type: {{.TypeLabelValue}}
+  egress:
+  - toEndpoints:
+    - matchLabels:
+        net-pol-test: {{.TargetLabelValue}}
+{{if .OnlyTargetNamespace}}
+        k8s:io.kubernetes.pod.namespace: {{.TargetNamespace}}
+{{end}}
+{{if .L7Enabled}}
+    toPorts:
+    - ports:
+      - port: "{{.TargetPort}}"
+        protocol: TCP
+      rules:
+        http:
+        - method: GET
+          path: /
+{{end}}
+{{else if .NetworkPolicy_Type "ccnp" }}
```

Comment:
I don't quite follow the if else here. Can you use indentation those to make the condition flow clearer?